.github/workflows/create-release-tag.yaml

```diff
@@ -0,0 +1,60 @@
+name: tag-generation
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+    paths:
+      - 'CHANGELOG.md'
+
+jobs:
```

Comment:
You probably want this job to depend on others, like tests and audit so that they can block the release if failing

Reply:
I'm not sure what you mean, this action runs only after a PR is merged to main, which would require the tests and audits to have already passed. This workflow runs before `release.yaml` to kick off that action, which already doesn't depend on tests or audit for the above reason